stolon/templates/keeper-ro-service.yaml

```diff
@@ -3,11 +3,10 @@ kind: Service
 metadata:
   name: {{ template "stolon.keeper.fullname" . }}
   labels:
-    app: {{ template "stolon.keeper.fullname" . }}
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: "stolon-keeper"
     release: "{{ .Release.Name }}"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     heritage: "{{ .Release.Service }}"
-    component: keeper
```

Comment:
Please, keep `component` label in place, just set it to the same value as current `app`.

Hopefully your RFE will be addressed inside the stolon repo, and we could reverse app label later. Essentially it is more logical to have app label corresponding to the fullname and component to, well, component.

Reply:
Why should it be there? It's only useful on pods right? 

I intentionally removed component + stolon-cluster labels from all spec excluding pods, as well as I removed chart and heritage from pods in order to not have all labels doubled in the chart templates. 

Maybe I'm missing something

Reply:
If this is not intended I add component and the stolon-cluster label to all labels sections again.

Reply:
![image](https://user-images.githubusercontent.com/5276857/38295943-17232994-37f0-11e8-96bc-c7fc4b1e9c1e.png)

Reply:
![image](https://user-images.githubusercontent.com/5276857/38295972-31d7e734-37f0-11e8-8085-edd894725ad9.png)

![image](https://user-images.githubusercontent.com/5276857/38296024-5255a7ee-37f0-11e8-997c-a625c370f8a3.png)

Reply:
![image](https://user-images.githubusercontent.com/5276857/38296043-67296188-37f0-11e8-8ee0-aeafefbde195.png)